server/config/config_test.go

```diff
@@ -196,3 +196,63 @@ func newTestScheduleOption() (*ScheduleOption, error) {
 	opt := NewScheduleOption(cfg)
 	return opt, nil
 }
+
+func (s *testConfigSuite) TestDefaultStoreLimitUpgrade(c *C) {
```

Comment:
do we need to add test set old and new name through `pd-ctl`?

Reply:
It is manually tested...

Reply:
Anyway， I think it is a good idea, I will add the test cases.

Reply:
test cases added, PTAL